src/java.base/share/classes/java/text/DecimalFormatSymbols.java

```diff
@@ -188,6 +188,15 @@ public static final DecimalFormatSymbols getInstance(Locale locale) {
         return dfsyms;
     }
 
+    /**
+     * {@return locale used to create this instance}
```

Comment:
Hello Jim, the opening and closing `{`, I think aren't needed for a `@return`

Reply:
This is actually a feature of JavaDoc. Accessor methods that require little description (self evident) can use {@return ...} to define the description and return in one go.

Reply:
Didn't know that, thank you for clarifying.